crates/ruff_linter/src/source_kind.rs

```diff
@@ -22,10 +26,19 @@ impl SourceKind {
         }
     }
 
+    /// Returns the source code for this source kind.
```

Comment:
```suggestion
    /// Returns the python code for this source kind.
```
I'm trying to make it obvious that we're not talking about the json as source code of a jupyter notebook. I thought about "extracted" but that doesn't make sense for a plain text file.